chrome/updater/updater_module.cc

```diff
@@ -98,17 +98,23 @@ void Observer::OnEvent(Events event, const std::string& id) {
       status = "Status is unknown.";
     } else if (crx_update_item_.state == ComponentState::kUpToDate &&
                updater_configurator_->GetPreviousUpdaterStatus().compare(
-                   updater_status_string_map.find(UpdaterStatus::kUpdated)
-                       ->second) == 0) {
+                   updater_status_string_map.at(UpdaterStatus::kUpdated)) == 0) {
       status = std::string(
-          updater_status_string_map.find(UpdaterStatus::kUpdated)->second);
+          updater_status_string_map.at(UpdaterStatus::kUpdated));
     } else {
       status = std::string(
           updater_status_string_map.find(status_iterator->second)->second);
     }
     if (crx_update_item_.state == ComponentState::kUpdateError) {
-      status +=
-          ", error code is " + std::to_string(crx_update_item_.error_code);
+      // QUICK_ROLL_FORWARD update, adjust the message to "Update installed,
+      // pending restart"
+      if (crx_update_item_.error_code == 21) {
```

Comment:
It took me a while, but is this 21 the one we're talking about?

https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:components/update_client/update_client_errors.h;l=74;drc=ac02e200288bef9c817fad9b06763463e4e9493d

If so, please use the enum. If that's not the 21, try to use the original value, and if that's not defined  ( maybe is from the OS), at least define a `constexpr int kBlaBla = 21`.

Reply:
Good point! Done.